### ISSUE TITLE
Your Avacode to me the whole system lays

### ISSUE THREAD

**Issue by guesshow** [avocode/avocode#1538]
IMPORTANT: Please try to describe how did it happen.

-----
App v. `2.14.6`
Avocode UID: 152102
```
Error: aborted
    at Uploader._uploadArtboardPath (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/uploader/lib/uploader2.js:451:31)
    at /Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:3018:46
    at List.__iterate (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:2208:13)
    at IndexedIterable.mappedSequence.__iterateUncached (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:3017:23)
    at seqIterate (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:606:16)
    at IndexedIterable.IndexedSeq.__iterate (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:322:14)
    at IndexedIterable.toArray (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:4260:23)
    at new List (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:2067:62)
    at reify (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:3572:37)
    at List.map (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:4403:14)
    at /Applications/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/uploader/lib/uploader2.js:417:36
```

**Comment by connor-bean** [comment]
<img width="1440" alt="2017-07-18 15 41 32" src="https://user-images.githubusercontent.com/17878371/28317854-e1c0039c-6bc8-11e7-897a-fa677c789fe0.png">

**Comment by jakubzitny** [comment]
Hi @guesshow, thanks for reaching out. This is a bug, we will fix it till next release of Avocode and let you know here.

**Comment by jakubzitny** [comment]
Hi there, this should be fixed in the latest Avocode `2.14.7`.

Please let us know if you have any more problems with Avocode 😉
